package.json

```diff
@@ -46,7 +46,7 @@
     "test-soy": "echo Starting soy tests ... && karma start test/soy_tests.conf",
     "pretest-core-mode": "npm run build && npm run build-cwc-main",
     "test-core-mode": "karma start test/core_tests.conf && karma start test/mode_tests.conf",
-    "pretest-units": "node build/test/unit_files.js",
+    "pretest-units": "node build/test/unit_files.js && npm run build-cwc-frameworks",
```

Comment:
The units tests should be possible without building the frameworks.
In the future I will change the test system to deps.js so that no compilation is needed at all.

Reply:
deps.js would be good. As-is, I have to mark everything from src/ used by the preview test as @export. I made an attempt at integrating deps.js into core_tests.js but it was a bit more complicated than I have time for at the moment, so hopefully a few extra @exports are acceptable.

Note: a 2nd commit to this PR with that is not yet pushed, as I still need to refactor preview.executeScript() to return a promise.